lc-hax/Scripts/Commands/Privileged/LevelEnemiesCommand.cs

```diff
@@ -0,0 +1,19 @@
+using Hax;
+using System;
+using UnityEngine;
+
+[DebugCommand("/enemies")]
+public class LevelEnemiesCommand : ICommand {
+
+    public void Execute(StringArray _) {
+        if (Helper.RoundManager == null) return;
+        if (Helper.RoundManager.currentLevel == null) return;
+        string Msg = "";
+        foreach (System.Collections.Generic.KeyValuePair<string, GameObject> enemy in Helper.SpawnableEnemies) {
+            Msg += $"{enemy.Key}, ";
+        }
```

Comment:
Use `string.Join` instead.

Reply:
is the same!

Reply:
no it isn’t. string.Join only does the string concatenation once. your method does the concatenation N times.

Reply:
Yeah i fixed that.